### DISCUSSION TITLE
How long does github student plan to be approved?

### DISCUSSION THREAD

**Discussion by ihakand** [community/community#22081] [New to GitHub]
<p>I requested for github student plan in 14 july but I am still waiting.<br>
How long it takes?<br>
<img src="https://user-images.githubusercontent.com/48999739/181068813-c6054b33-9f12-44e5-991b-aca908b7cad4.png" alt="image" data-base62-sha1="4iqQrCqHbPYtRDEzB7Y35MtW0DE" width="392" height="147"></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>At present, I believe the education team is doing their best to have applications resolved within three weeks.</p>
<p>Sorry we can’t be more specific - some applications take longer than others.</p>
